test/test-remote/testutils/index.ts

```diff
@@ -154,9 +154,12 @@ export function globalTestSuiteSetupOnce() {
     TENANT_DEFAULT_API_TOKEN_FILEPATH
   ]) {
     if (tfp !== undefined) {
-      const token = fs.readFileSync(tfp, {
-        encoding: "utf8"
-      });
+      const token = fs
+        .readFileSync(tfp, {
+          encoding: "utf8"
+        })
+        .trim();
```

Comment:
Note: this makes it easier to manually create these token files for test-remote; a leading or trailing whitespace won't break things anymore.